### PR TITLE
Terminal Hangman

### LINKED ISSUE #793: Hangman on terminal
# Aim

- This will be a python script that can be used to play hangman on terminal. Unlike the one present right now. This script will be completely made importing single module "random".

# Details

The Script will have following feature
- Terminal art 
image
- Customizable words
- Complete game on command line interface.

### PR DESCRIPTION
**Pull request by Yuvraj-kadale**
# Description

Added Terminal Hangman

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: Terminal Hangman/Readme.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Terminal Hangman
+
+## How to play ?
+
+#### Easy to begin with, just run the code and terminal says it all.
```
Comment (kaustubhgupta) [inline Terminal Hangman/Readme.md:5]: Remove this line

File: Terminal Hangman/T_hangman.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import random
+
+
+def hangman():
+
+    word = random.choice(["python",
```
Comment (kaustubhgupta) [inline Terminal Hangman/T_hangman.py:6]: Store these words in a text file and load from it

File: Terminal Hangman/T_hangman.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                          "spiderman",
+                          "tiger",
+                          "superman",
+                          "thor",
+                          "pokemon",
+                          "avengers",
+                          "savewater",
+                          "earth",
+                          "annable"])                        # Word list we would like to play with.
+    validLetters = 'abcdefghijklmnopqrstuvwxyz'              # All the characters defined in the word list
```
Comment (kaustubhgupta) [inline Terminal Hangman/T_hangman.py:16]: Using the "string" module to get the alphabets rather than hardcoding them.

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Terminal Hangman/T_hangman.py [hunk 4]
```diff
@@ -0,0 +1,112 @@
+import random
+
+def Mywordlist():
+    '''
+    This function will fetch random words from my custom word list by opening file in read mode
+    '''
+    with open("data.txt", "r") as file:
```
Comment (kaustubhgupta) [inline Terminal Hangman/T_hangman.py:7]:
```suggestion
    with open("./Terminal Hangman/data.txt", "r") as file:
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by Yuvraj-kadale** [comment]
@kaustubhgupta btw both .txt and .py files are in same folder .. and the code was working.

**Review by santushtisharma10 — APPROVED** [review]
@Yuvraj-kadale 💯

**Comment by santushtisharma10** [comment]
@Kushal997-das

**Comment by santushtisharma10** [comment]
@antrikshmisri @Kushal997-das

**Review by kaustubhgupta — APPROVED** [review]

**Review by antrikshmisri — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM

**Pull request merged**